include/alpaka/rand/RandDefault.hpp

```diff
@@ -0,0 +1,198 @@
+/* Copyright 2021 Jeffrey Kelling
+ *
+ * This file is part of alpaka.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#pragma once
+
+#include <alpaka/core/Common.hpp>
+#include <alpaka/core/Unused.hpp>
+#include <alpaka/math/cos/Traits.hpp>
+#include <alpaka/math/log/Traits.hpp>
+#include <alpaka/math/sin/Traits.hpp>
+#include <alpaka/math/sqrt/Traits.hpp>
+#include <alpaka/rand/RandPhilox.hpp>
+#include <alpaka/rand/Traits.hpp>
+
+#include <limits>
+#include <type_traits>
+
+namespace alpaka
+{
+    namespace rand
+    {
+        class RandDefault : public concepts::Implements<ConceptRand, RandDefault>
+        {
+        };
+
+        namespace distribution
+        {
+            namespace gpu
+            {
+                namespace detail
+                {
+                    template<typename TFloat>
+                    struct BitsType;
+
+                    template<>
+                    struct BitsType<float>
+                    {
+                        typedef std::uint32_t type;
```

Comment:
```suggestion
                        using type = std::uint32_t;
```
In more places as well.